openlibrary/coverstore/archive.py

```diff
@@ -89,6 +91,50 @@ def close(self):
 idx = id
 
 
+def is_uploaded(item, f):
+    command = fr'ia list {item} | grep "{f}\.[tar|index]" | wc -l'
+    result = subprocess.run(command, shell=True, capture_output=True)
+    output = result.stdout.decode().strip()
+    return int(output) == 2
+
+
+def audit(group_id, chunk_ids=(0, 100), sizes=('', 's', 'm', 'l'), upload=False):
```

Comment:
Rm these; they aren't used.
```suggestion
def audit(chunk_ids=(0, 100), sizes=('', 's', 'm', 'l')):
```